game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -6,7 +6,33 @@ package io.aigar.game
  * of the players.
  */
 class GameThread extends Runnable {
+  val MillisecondsPerTick = 16
```

Comment:
`final private`? According to http://stackoverflow.com/a/13412894 it could be good for us